app/router/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const appRouter = require('./routers');
+const middlewares = require('../../middlewares/middlewares');
+const mainController = require('../controllers/mainController');
+
+const router = new express.Router();
```

Comment:
```suggestion
const router = express.Router();
```

Pas besoin du `new` ici